assets/sass/components/adminbar/_googlesitekit-adminbar-overrides.scss

```diff
@@ -143,6 +143,22 @@
 			@extend .mdc-layout-grid__cell--span-8-tablet;
 		}
 
+		// Button
+		.mdc-button {
+
+			@extend .mdc-button;
+		}
+
+		.mdc-button--raised {
+
+			@extend .mdc-button--raised;
+		}
+
+		.mdc-button__label {
+
+			@extend .mdc-button__label;
```

Comment:
I'm not sure if I am missing something, but this `@extend` doesn't seem to be adding any styles when trying it out locally. Also, the button is missing some styling - for example it should be `text-transform: uppercase` - but the default `#wpadminbar > *` styles are overriding it.

![image](https://user-images.githubusercontent.com/18395600/159971310-d65dd968-1617-445b-a03b-4eabe2606b04.png)

Note how there are no styles applied to the `mdc-button__label`, and the `span` is inheriting from `mdc-button--raised` / `mdc-button` and being overriden by `#wpadminbar > *`:
![image](https://user-images.githubusercontent.com/18395600/159972973-3d40cd8d-2b85-4c19-8788-445e07f7da97.png)

Looks like something needs addressing here - do you get the same running locally?

Reply:
As mentioned, I kind of went for the Figma mock here but nice catch on the label not taking any effect! I kind of know what is happening here and I discussed it with Matthew earlier today. I have overridden the styles to make the CTA look like other CTAs in the app as per [this discussion](https://github.com/google/site-kit-wp/pull/4862#discussion_r811018503) that you refer too.

Reply:
I have kept the font size 13px (instead of 14px) and the weight to 400 instead of 500 cause having a super chunky button in the Admin Bar where everything else is smaller feels out of place to me.